.github/workflows/ci.yml

```diff
@@ -1,6 +1,9 @@
 name: CI
 
-on: [push]
+on:
+  push:
+    branches:    
+      - master
```

Comment:
But now we lost a pipeline for non-master pushes.
My idea was to leave `ci.yml` as it was, and create a separate pipelines for releases. For instance, `pre-release.yml` that would create pre-releases on every push to master, and `release.yml` that would create releases when manually triggered.

Reply:
In that case, can't we just rename the current `ci.yml` to `pre-release.yml` and then revert to the original `ci.yml`? The `release.yml` seems to be mostly orthogonal, no?

Reply:
I see a couple of differences in the pipelines: In the pre-release and release pipelines we don't need windows in the OS matrix. And in the release pipline the version must be taken from the package.json, not created based on the current date/time. Later, the release pipeline can be expanded to do real publishing to the marketplace and to Open VSX.